crates/matrix-sdk-ui/tests/integration/timeline/replies.rs

```diff
@@ -107,16 +108,13 @@ async fn in_reply_to_details() {
     let _response = client.sync_once(sync_settings.clone()).await.unwrap();
     server.reset().await;
 
-    let _read_receipt_update =
-        assert_matches!(timeline_stream.next().await, Some(VectorDiff::Set { value, .. }) => value);
-
-    let third = assert_matches!(
-        timeline_stream.next().await,
-        Some(VectorDiff::PushBack { value }) => value
+    assert_let!(
+        Some(VectorDiff::Set { value: _read_receipt_update, .. }) = timeline_stream.next().await
     );
```

Comment:
Looks like this doesn't use the binding, so could just be `assert_matches!` or even `assert!(matches!(...))`?

Reply:
I had that in a previous revision, but found it made the test less clear. The right thing to do would be to add an assert that it's actually a read receipt update.